nvflare/app_opt/xgboost/histogram_based_v2/adaptors/grpc_client_adaptor.py

```diff
@@ -99,6 +102,12 @@ def start(self, fl_ctx: FLContext):
         if not self.num_rounds:
```

Comment:
can "self.num_rounds" be set to "0"?
If not allowed,  should we change the error msg to something like:
```
raise RuntimeError("cannot start - num_rounds is not set or set to 0")
```

Reply:
This one is removed and added the > 0 check in xgb_adapter.py.